tests/Fakes/DataWithMergedRuleset.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Spatie\LaravelData\Tests\Fakes;
+
+use Spatie\LaravelData\Attributes\MergeRules;
+use Spatie\LaravelData\Attributes\Validation\Max;
+use Spatie\LaravelData\Data;
+
+#[MergeRules]
+class DataWithMergedRuleset extends Data
+{
+    public function __construct(
+        #[Max(10)]
+        public string $first_name,
+    ) {
+    }
+
+    public static function rules(): array
+    {
+        return [
+            'first_name' => ['min:2']
+        ];
```

Comment:
Inline these classes within the tests, that makes the fakes directory a bit less heavy.

Reply:
Have sorted this, and have added a new test to check validation rules are skipped using `WithoutValidation`